lib/sqlalchemy/dialects/mssql/base.py

```diff
@@ -2585,15 +2586,37 @@ def get_view_definition(
     def get_columns(self, connection, tablename, dbname, owner, schema, **kw):
         # Get base columns
         columns = ischema.columns
+        computed_cols = ischema.computed_columns
         if owner:
             whereclause = sql.and_(
                 columns.c.table_name == tablename,
                 columns.c.table_schema == owner,
             )
+            table_fullname = "%s.%s" % (owner, tablename)
+            concat = func.concat(
+                columns.c.table_schema, ".", columns.c.table_name
+            )
+            join_on = computed_cols.c.object_id == func.object_id(concat)
```

Comment:
I'm not sure if this is the best solution, but I have not found the object id of the table in the information_schema.column table